src/sql/workbench/contrib/notebook/browser/htmlMarkdownConverter.ts

```diff
@@ -150,3 +219,20 @@ export function findPathRelativeToContent(notebookFolder: string, contentPath: U
 	}
 	return '';
 }
+
+export function escapeAngleBrackets(textContent: string, node?) {
+	let text: string = textContent;
+	if (text.includes('<u>') || text.includes('<mark>') || (text.includes('style') && !text.includes('<style>'))) {
+		return text;
+	}
+	let mapTags = { '<': '\\<', '>': '\\>' };
+
+	let escapedText = text.replace(/<|>/gi, function (matched) {
+		return mapTags[matched];
+	});
+	return escapedText;
+}
+
+function repeat(character, count) {
```

Comment:
Return type for the function

Reply:
refactored the repeat function, to use string.repeat instead.